test/controllers/address_controller_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class AddressControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get address_index_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get address_new_url
+    assert_response :success
+  end
+
+  test "should get create" do
+    get address_create_url
```

Comment:
you should test that something is actually created/deleted/fetched/etc correctly not just that the urls exist